### ISSUE TITLE
POST request with FromData file uploading is blocked

### ISSUE THREAD

**Issue by yuriMalakhov** [mukuljainx/Mokku#22]
@mukuljainx Hey, thank you for the great extension! It is really useful.

It seems I have found an issue when file uploading requests are blocked somewhere internally in the extension and not even displayed neither in the Network tab nor in the extension logs.

Disabling the extension makes file uploading to work again.

Below is a copy of a fetch request that is blocked:
```
fetch("http://localhost:3000/property/3ea8d52b-25e1-44e3-93a3-a4edd6f94cfe/key/image", {
  "headers": {
    "accept": "application/json, text/plain, */*",
    "accept-language": "en-GB,en-US;q=0.9,en;q=0.8",
    "authorization": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIwLjAuMC4wIiwiaXNzIjoibG9jYWxfa2lrb19hdXRoIiwiZXhwIjoxNjAxMjA2MTYzfQ.sKTTuka_MrULJD-5aSDDDF5kMwTS-iZAxYk91UhVB0g",
    "cache-control": "no-cache",
    "content-type": "multipart/form-data; boundary=----WebKitFormBoundaryv1nRBcZ21uzVdy17",
    "pragma": "no-cache",
    "proxy-connection": "keep-alive",
    "sec-fetch-dest": "empty",
    "sec-fetch-mode": "cors",
    "sec-fetch-site": "same-site"
  },
  "referrer": "http://localhost:4140",
  "referrerPolicy": "no-referrer-when-downgrade",
  "body": "------WebKitFormBoundaryv1nRBcZ21uzVdy17\r\nContent-Disposition: form-data; name=\"file\"; filename=\"freddi.jpg\"\r\nContent-Type: image/jpeg\r\n\r\n\r\n------WebKitFormBoundaryv1nRBcZ21uzVdy17--\r\n",
  "method": "POST",
  "mode": "cors",
  "credentials": "include"
});
```

**Comment by mukuljainx** [comment]
Hi @yuriMalakhov, Thanks for reporting the issue & the detailed description, I have started working on it will let you know as this is fixed.

**Comment by mukuljainx** [comment]
Hi @yuriMalakhov This has been fixed with the commit  [fix(inject): change request body to string in case of object-upload](https://github.com/mukuljainx/Mokku/commit/7776de44e36735bd421c276856c6fc06c9c5598d), I have submitted it for review in the store, should be published by tomorrow as 1.0.7

This should fix the bug I tried to upload the sample file (code is in example directory) and was I able to identify the issue, if doesn't please let me know.
